### PR TITLE
Initial Setup

### LINKED ISSUE #45: Stock Viewer
**Is your feature request related to a problem? Please describe.**
It is quite a headache to go to the website and view today's stock situations currency, bitcoin values, etc.

**Describe the solution you'd like**
I want to build an extension where one can get news of current  INR to dollar values top Stock prices and bitcoin values in just one click

### PR DESCRIPTION
**Pull request by shivanshsin0203**
Fixes #45 

Created Initial Popup Html and Css files of extension

image

Screenshot 2024-01-06 210301

### CODE REVIEW

**Comment by shivanshsin0203** [comment]
Hey @nik132-eng made initial setup

**Comment by nik132-eng** [comment]
Add more features in upcoming commit

**Comment by shivanshsin0203** [comment]
Hey @nik132-eng I have created initial popup files and will be adding functional js in the comming 2-3 days

**Review by nik132-eng — APPROVED** [review]

File: Score Viewer/logo.png
[the remark this answers is not in the document]
Comment (shivanshsin0203) [reply Score Viewer/logo.png:1]: I have added it in manifest.json

**Pull request merged**